Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -1439,4 +1439,9 @@ public boolean isDirectHatAllowed() {
     public boolean isConfirmCommandEnabledByDefault(String commandName) {
         return getDefaultEnabledConfirmCommands().contains(commandName.toLowerCase());
     }
+
+    @Override
+    public boolean isTeleportBackWhenFreedFromJail() {
+        return config.getBoolean("teleport-back-when-freed-from-jail", true);
```

Comment:
Could you alter this to be consistent with most pre-existing Essentials settings, only reading from the config once and returning the stored value until the config is reloaded?

Reply:
Fixed.